### ISSUE TITLE
Iranians in emergency, connecting/syncing terminal to app

### ISSUE THREAD

**Issue by Shahin-rmz** [simplex-chat/simplex-chat#1101]
Hello, Thanks for providing fast and secure app to make people connect with each other, without disrespecting one's privacy. 
For Iranian Users this is a **temporary** good solution. once the government gets aware about simplex, they will ban it completely.
Now a question, how can I connect Simplex of my terminal, to android app?
Thanks.

**Comment by mika-nl** [comment]
Is tor and bridges blocked in Iran ?
Because you can use simplex-chat over tor.

It is also easy to setup a SMP server. 
When it is not public it is not so easy to block

**Comment by Shahin-rmz** [comment]
> Is tor and bridges blocked in Iran ?




Yes, but recently Anonymous declared that there are new methods of using Tor.
Although Tor bridges are blocked, but recently people can connect to it.
Also I am trying to make simplex popular between my friends.

**Comment by ElsAr4e** [comment]
@Shahin-rmz You could also try P2P-messengers like Jami https://jami.net/ to see if they are also blocked.
